scannerlib/proto/scan_instructions.proto

```diff
@@ -272,4 +272,6 @@ message SQLCheck {
   string non_compliance_msg = 4;
   // Only needed for ElasticSearch database, perform regex match on response.
   string filter_regex = 5;
+  // (Optional) the query result
```

Comment:
What is this used for? I don't see it referenced anywhere in the code.

Reply:
It was for an initial implementation and was committed by mistake. Removed.